var/spack/repos/builtin/packages/compiler-wrapper/package.py

```diff
@@ -43,7 +43,7 @@ class CompilerWrapper(Package):
     if sys.platform != "win32":
         version(
             "1.0",
```

Comment:
This is just my curiosity, but how is this versioned when decoupled from spack core?